dmscripts/generate_questions_csv.py

```diff
@@ -95,9 +118,16 @@ def generate_csv(output_directory, framework_slug, content_loader, context):
     for index, row in enumerate(rows):
         row.extend([''] * (max_length - len(row)))
 
-    with open('{}/{}-questions.csv'.format(output_directory, framework_slug), 'wb') as csvfile:
-        writer = csv.writer(csvfile, delimiter=b',', quotechar=b'"')
-        header = ["Page title", "Page title hint", "Question", "Hint"]
+    filename = framework_slug
```

Comment:
None of this `filename` stuff gets used at all now as the `output_file` is passed in, so this section can probably be deleted. 

(Although I like the idea of giving the script a directory where it will generate sensibly-named files for you depending on the params you pass in, so would be happy to see that fleshed out too as an alternative to deleting this `filename` stuff.)